src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-document-card/index.js

```diff
@@ -480,5 +499,14 @@ export default {
                 this.onPrepareDocument();
             }
         },
+
+        availableFormatsFilter(item) {
+            const fileTypesArray = Object.values({ ...item.config?.fileTypes });
+            if (fileTypesArray.includes('pdf') && fileTypesArray.includes('html')) {
+                return fileTypesArray.join(', ').toUpperCase();
+            }
+
+            return this.$tc('sw-order.documentCard.labelOnlyPdf');
```

Comment:
This might work for now, but I still don't like it because if somehow `PDF` or `HTML` are missing, but it's a different type e.g. `XML`, it would still say `Only PDF` which is wrong. And changing this could be easily forgotten in the future.

I would build it slightly more future proof:
```suggestion
            let label = '';
            if (fileTypesArray.length === 1) {
                label += this.$tc('sw-order.documentCard.labelOnly') + ' '; // Snippet should just say "Only" translated
            }

            return label + fileTypesArray.join(', ').toUpperCase();
```

Reply:
This is the main reason I think the demo'd way is better / neater:

![image](https://github.com/user-attachments/assets/98af46ba-27b8-4493-ad0d-d6b99242cbcc)

Reply:
@Bird87ZA I also like this UI, I will create a ticket to improve it later

Reply:
Already have one NEXT-40357, and still in discussion